core/chain.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * @module core/chain
+ */
+'use strict';
+
+const chainData = require('core/chaindata');
+
+/**
+ * Returns balance of account.
```

Comment:
For return value there is **@return** (or **@returns** tag) in jsdoc. It's a good practice to describe what function does in a more generic way.